docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -0,0 +1,3023 @@
+OpenMS User Tutorial
+====================
+
+## General Remarks
+
+- This handout will guide you through an introductory tutorial for the OpenMS/TOPP software package[^1].
+
+- OpenMS[^2]<sup>,</sup>[^3] is a versatile open-source library for mass spectrometry data analysis. Based on this library, we offer a collection of command-line tools ready to be used by end users. These so-called TOPP tools (short for ”The OpenMS Pipeline”)[^4] can be understood as small building blocks of arbitrarily complex data analysis workflows.
+
+- In order to facilitate workflow construction, OpenMS was integrated into KNIME[^5], the Konstanz Information Miner, an open-source integration platform providing a powerful and flexible workflow system combined with advanced data analytics, visualization, and report capabilities. Raw MS data as well as the results of data processing using TOPP can be visualized using TOPPView[^6].
+
+- This tutorial was designed for use in a hands-on tutorial session but can also be worked through at home using the online resources. You will become familiar with some of the basic functionalities of OpenMS/TOPP, TOPPView, as well as KNIME and learn how to use a selection of TOPP tools used in the tutorial workflows.
+
+- All sample data referenced in this tutorial can be found in the
+`C: / Example_Data` folder, on the USB stick that came with this tutorial, or released online on our GitHub repository `OpenMS/Tutorials`.
```

Comment:
btw. will this webpage be used in our tutorial sessions from now on, instead of the usual PDF?
Just want to make sure we really want these mentions of a "hands-on tutorial sessions" that few people can attend and a "USB stick" on this online public facing tutorial page.

Reply:
yes this is the idea.
I agree we should at least add a sentence "if you are attending the tutorial and received a USB stick"

Reply:
Done.